components/inputs/input-number.js

```diff
@@ -147,6 +147,11 @@ class InputNumber extends FormElementMixin(LocalizeCoreElement(LitElement)) {
 		this._formattedValue = value;
 		await this.updateComplete;
 		this.value = parseNumber(value);
+
+		this.dispatchEvent(new CustomEvent(
+			'change',
+			{ bubbles: true, composed: false }
+		));
```

Comment:
Could you add some tests for this as well? We want to make sure that if the underlying value didn't actually change (even though the value the user typed in might have), that we don't fire the event.